3p/integration.js

```diff
@@ -145,6 +145,7 @@ import {improvedigital} from '#ads/vendors/improvedigital';
 import {inmobi} from '#ads/vendors/inmobi';
 import {innity} from '#ads/vendors/innity';
 import {insticator} from '#ads/vendors/insticator';
+import {insurads} from '#ads/vendors/insurads';
```

Comment:
No need to change this file. It was for backwards compatibility. Please revert them.

Reply:
Done 👍